app/javascript/controllers/components/App.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+
+function App () {
+  return (
+    <div>My app is awesome!</div>
```

Comment:
- [ ] The App component should include routes from `react-router`, kindly make sure to add this so the app is working properly

Reply:
done

![image](https://user-images.githubusercontent.com/92118813/211975041-540574c8-f875-44f2-ad9c-ae868e043446.png)